### PR TITLE
feat: bump react native image picker enabling multiple image selection

### PR DESCRIPTION
**Pull request by beneditomauro**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-robin-revamp/issues/252 |

### CODE REVIEW

File: react/util/ui-utils.js [hunk 1]
```diff
@@ -98,19 +98,21 @@ export const pickImageCamera = async function ({
 
 export const pickImageGalery = async function ({
     mediaType = "photo",
+    selectionLimit = 5,
```
Comment (gcandal) [inline react/util/ui-utils.js:101]: Is this required?

Comment (beneditomauro) [reply react/util/ui-utils.js:101]: @gcandal yes, but maybe I should set to 1
ertyup

Comment (gcandal) [reply react/util/ui-utils.js:101]: Go with 0 then

File: react/util/ui-utils.js [hunk 2]
```diff
@@ -59,15 +59,15 @@ export const pickDocuments = async function (options = {}) {
     }
 };
 
-const normalizeImage = function (image) {
-    return {
+const normalizeImage = function (images) {
```
Comment (gcandal) [inline react/util/ui-utils.js:62]:
```suggestion
const normalizeImages = function (images) {
```

**Review by gcandal — COMMENTED** [review]
Don't forget the changelog entry

**Review by gcandal — APPROVED** [review]

**Review by joamag — APPROVED** [review]

**Pull request merged**